prediction-polls/android/app/src/main/java/com/bounswe/predictionpolls/utils/NavItem.kt

```diff
@@ -0,0 +1,37 @@
+package com.bounswe.predictionpolls.utils
```

Comment:
sealed class DrawerItem(val route: String, @StringRes val resourceId: Int, @DrawableRes val iconId: Int) {
    object Profile : DrawerItem("profile", R.string.nav_drawer_profile, R.drawable.ic_profile)
    object Feed : DrawerItem("feed", R.string.nav_drawer_feed, R.drawable.ic_feed)
    object VotePoll : DrawerItem("vote", R.string.nav_drawer_vote, R.drawable.ic_vote)
    object CreatePoll : DrawerItem("create_poll", R.string.nav_drawer_create, R.drawable.ic_create)
    object Moderation : DrawerItem("moderation", R.string.nav_drawer_moderation, R.drawable.ic_moderation)
    object Leaderboard : DrawerItem("leaderboard", R.string.nav_drawer_leaderboard, R.drawable.ic_leaderboard)
    object Notifications : DrawerItem("notifications", R.string.nav_drawer_notifications, R.drawable.ic_notifications)
}

Seems like a better solution to me. Better naming, and more concise

Reply:
I want to use this Item also in main navigation so to the name is not suitable however approach is better I will switch to this.